tests/foreman/cli/test_report.py

```diff
@@ -23,67 +23,63 @@
 from robottelo import ssh
 from robottelo.cli.base import CLIReturnCodeError
 from robottelo.cli.report import Report
-from robottelo.test import CLITestCase
 
 
-class ReportTestCase(CLITestCase):
-    """Test class for Reports CLI. """
+@pytest.fixture(scope='module')
```

Comment:
You could make this an auto use fixture and then remove it from each tests argspec.

Reply:
updated the pr.